src/CitizenProject/CitizenProjectListener.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace AppBundle\CitizenProject;
+
+use AppBundle\Entity\CitizenProject;
+use Doctrine\ORM\Event\LifecycleEventArgs;
+use League\Flysystem\Filesystem;
+use League\Glide\Server;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+class CitizenProjectListener
```

Comment:
I personally don't like listeners in most cases (specially doctrine listeners) because they make the code harder to understand.

I noticed that we don't any doctrine listeners yet on the project. 

Why do we need this one ?